### PR TITLE
Support extended tokens + expose token feature for library use

### LINKED ISSUE #10: Create usage documentation guide
We need to update the ReadME to contain comprehensive usage guide and clean up instructions not related to the package.

### PR DESCRIPTION
**Pull request by beesaferoot**
I updated some sample tokens as they did not match the expected tokens which I verified using the original python library. 

@dmohns You can confirm if the changes are accurate. Closes #10

### CODE REVIEW

**Comment by dmohns** [comment]
Hey @beesaferoot 

following up on my earlier comments. I think the issue was that we never actually had Extended Tokens in the test JSON 😅 I generated some using the legacy Library.

Could you add the JSON as per this PR: https://github.com/EnAccess/OpenPAYGO-js/pull/34 to your PR? (Probably you can just cherry pick my commit) And see if this resolves this issue?

File: test/encoder.test.js [hunk 1]
```diff
@@ -1,29 +1,25 @@
...
-        const { finalToken } = encoder.generateToken({
-          tokenType: data.token_type,
-          secretKeyHex: data.key,
-          count: data.token_count,
-          startingCode: data.starting_code,
-          restrictDigitSet: data.restricted_digit_set,
-          value: data.value_raw,
-          extendToken: false,
-        })
+      const useExtended = data.value_raw > shared.MAX_ACTIVATION_VALUE
```
Comment (dmohns) [inline test/encoder.test.js:11]: We should use this information from the newly added test cases. If the value is higher than allowed MAX_ACTIVATION_VALUE token generation should fail.

**Comment by devjhane** [comment]
Hello @dmohns  I got around this and updated the sample tokens from your PR. 

Everything seems to be working as expected now (for standard and extended token generation).

File: test/encoder.test.js (hunk 1 above)
Comment (devjhane) [reply test/encoder.test.js]: Added a assertion on the test to ensure this.

Comment (beesaferoot) [reply test/encoder.test.js]: Added a assertion on the test to ensure this.

**Comment by beesaferoot** [comment]
Hello @dmohns I got around this and updated the sample tokens from your PR.

Everything seems to be working as expected now (for standard and extended token generation).

**Review by dmohns — APPROVED** [review]

**Comment by devjhane** [comment]
Using #35  We can cross check the implementation, will look into that next. 

Thanks for merging 👍🏾

**Pull request merged**